_static/css/styles.css

```diff
@@ -1,4 +1,9 @@
 
 .navbar-header-items .navbar-header-items__center {
 	margin-left: 2em !important;
+}
```

Comment:
@sneakers-the-rat this is awesome. If you remember, let's try to add these changes to our theme as well! i think they are valuable in all of our books!!